rpn.py

```diff
@@ -0,0 +1,37 @@
+import re
+from collections import deque
+# operands, operators = [], []
+
+print('Reverse Polish Notation\n')
+
+expression = input("Enter a mathematical expression :\n").split()
+print(expression)
+# operands = [re.findall(r'\d+', expression)]
+# operators = [re.findall(r'\D+', expression)]
+
+#function to evalute the first operation
```

Comment:
@wassim201  It seems you already opened a [ first PR](#1 ) that got the same content. This new pr presents very slight changes (***line 12***). In this case it is required you drop the first PR since this one is the most updated version of your contribution.
Also, please put your files in dedicated folder with the topic of the contribution with a README.md explaining the use of the scripts ([see contribution example](#6 ))